### PR TITLE
Add Test on elevation processing and feature2Texture

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
* Add Test on elevation processing and feature2Texture.
* Remove test coverage on ThreeExtented

### CODE REVIEW

File: test/unit/dataSourceProvider.js [hunk 1]
```diff
@@ -12,24 +12,40 @@ import WMSSource from 'Source/WMSSource';
...
 import Feature2Mesh from 'Converter/Feature2Mesh';
-import MaterialLayer from 'Renderer/MaterialLayer';
+import LayeredMaterial from 'Renderer/LayeredMaterial';
 
 const holes = require('../data/geojson/holes.geojson.json');
 
 supportedFetchers.set('image/png', () => Promise.resolve(new THREE.Texture()));
 supportedFetchers.set('application/json', () => Promise.resolve(holes));
 
+global.document = {
```
Comment (zarov) [inline test/unit/dataSourceProvider.js:28]: I don't get it, can you explain ?

**Review by zarov — COMMENTED** [review]
Thanks, I have two things to ask:
- what is the need for the coveralls file ? It was working correctly before, no ?
- please append to your commit why you are removing the `ThreeExtended` folder from `nyc` (even if I know for myself why)

File: test/unit/dataSourceProvider.js (hunk 1 above)
Comment (gchoqueux) [reply test/unit/dataSourceProvider.js:28]: to Mock document object for Mocha.

**Comment by gchoqueux** [comment]
> What is the need for the coveralls file ? It was working correctly before, no ?

Coveralls file removed and coveralls status check recovered. 

> please append to your commit why you are removing the ThreeExtended folder from nyc (even if I 
know for myself why)

`ThreeExtended` isn't iTowns code, this's falsifying the code cover. Furthermore, it's difficult to mock fetching. I have add just a unit test on `gltfLoader`.

**Review by zarov — APPROVED** [review]
Nice to have a GLTF test ! Thanks for all

**Pull request merged**